examples/cloud-router-2-gcp-connection/outputs.tf

```diff
@@ -1,3 +1,12 @@
+output "GCP_Network_Id" {
+  value = google_compute_network.port-gcp.id
+}
+output "GCP_Router_Id" {
+  value = google_compute_router.port-gcp.id
+}
+output "GCP_Interconnect_Id" {
+  value = google_compute_interconnect_attachment.port-gcp.id
+}
 output "module_output" {
```

Comment:
Let's change this output name to `google_connection_id` to be consistent with the port example.